src/cli-validator/runValidator.js

```diff
@@ -230,12 +231,23 @@ const processInput = async function(program) {
       exitCode = 1;
       continue;
     }
-
+    // the warning property tells the user if warnings are included as part of the output
+    // if errorsOnly is true, only errors will be returned, so need to force this to false
+    if (errorsOnly) {
+      results.warning = false;
+    }
```

Comment:
Add new line after this block
```suggestion
    }

```